recipes-openxt/argo-exec/argo-exec/argo-input-sender

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+event=$1
+domain=0
+port=7777
+
+exec argo-exec -d "$domain" -p "$port" -- input-proxy-sender /dev/input/${event} 2>&1 | logger -t argo-input-sender &
```

Comment:
Took a look through the qubes sender, wouldn't it have been possible to replace their usage of stdin/stdout with a socket instead, and then we could run input-proxy-sender/receiver with ```LD_PRELOAD=/usr/lib/libargo-1.0.so INET_IS_ARGO=1``` to make use of the interposer?  This would be instead of introducing the argo-exec.c.  Not that argo-exec is a bad implementation or anything, just the overall code added should be less, I imagine.  Plus the interposer has been in use for awhile and therefore has some implicit testing.

Reply:
Qubes RPC runs commands between VMs with stdin & stdout wired up, and input-proxy-sender/receiver are written to use that.  I wrote argo-exec to provide that same execution environment.

While we could patch input-proxy-sender/receiver, I'd prefer not to.  I can't recall precisely, but I think I chose argo-exec to avoid adding patches.  Every patch we carry is more work when uprev-ing.  (Were I to patch it, I'd use argo non-interposer since I'd prefer elimination of LD_PRELOAD)

It's generic enough that it could be re-used for something else, though I don't have anything in mind.

Reply:
Ok, I follow.  Their stdin/stdout implementation seems strange to me though, but if it works it works.